### PR TITLE
Remove update statistics

### LINKED ISSUE #149: [Refactor] remove UpdateStatistics
With https://github.com/DroneDB/DroneDB/pull/220 both `objectsCount` and `size` are simply part of the database info, so UpdateStatistics could/should be removed.

### PR DESCRIPTION
**Pull request by HeDo88TH**
Closes #149

### CODE REVIEW

File: Registry.Web/Utilities/Extenders.cs [hunk 1]
```diff
@@ -81,8 +79,8 @@ public static DatasetDto ToDto(this Dataset dataset, DdbAttributes attributes)
                 LastEdit = attributes.LastUpdate,
                 Name = dataset.Name,
                 Meta = attributes.Meta,
-                ObjectsCount = dataset.ObjectsCount,
-                Size = dataset.Size,
+                ObjectsCount = attributes.ObjectsCount,
+                //Size = dataset.Size,
```
Comment (pierotofy) [inline Registry.Web/Utilities/Extenders.cs:83]: Did you mean to comment this out?

**Comment by pierotofy** [comment]
See the one comment I left; otherwise go ahead and merge!

**Comment by HeDo88TH** [comment]
> See the one comment I left; otherwise go ahead and merge!

You can review it now. That line slipped through the cracks 👍

File: Registry.Web/Utilities/Extenders.cs (hunk 1 above)
Comment (HeDo88TH) [reply Registry.Web/Utilities/Extenders.cs]: Fixed in https://github.com/DroneDB/Registry/pull/155/commits/a151580085ce2410b77f505220e8bd89aabc8496

**Review by pierotofy — APPROVED** [review]

**Pull request merged**